rebuild-initrd.yml

```diff
@@ -21,17 +21,25 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
+#!/usr/bin/env ansible-playbook
 ---
-dependencies: []
+- name: setup cfs_image host group
+  gather_facts: false
+  hosts: all
+  tasks:
+    - name: Add image customization hosts to the cfs_image host group.
+      add_host:
+        name: '{{ inventory_hostname }}'
+        groups: cfs_image
+      when: cray_cfs_image
 
-galaxy_info:
-  role_name: csm.password
-  author: randy.kleinman@hpe.com
-  description: Password management for CSM users
-  company: "Hewlett Packard Enterprise Development LP"
-  license: "MIT"
-  min_ansible_version: 2.9
-  platforms: []
-  galaxy_tags:
-    - system
-    - hpc
\ No newline at end of file
+- hosts: Application:&cfs_image
+  gather_facts: true
```

Comment:
Here is a great example of an opportunity for using "gather_subset".  See https://github.com/Cray-HPE/csm-config/commit/8f23e8049827269c191b43dac246b051b1580ca0 for an example of the change we made to our repo specifically for a task that required "ansible_distribution"

Reply:
I opted to remove this playbook and turn it into a role.